src/scheduler/data_types.h

```diff
@@ -1098,6 +1097,7 @@ struct config
 	char **res_to_check;			/* the resources schedule on */
 	resdef **resdef_to_check;		/* the res to schedule on in def form */
 	char **ignore_res;			/* resources - unset implies infinite */
+	char *job_sort_formula;			/* job sort formula for the sched */
```

Comment:
This is yet another way to hold data from the scheduler object.  Some are in globals by themselves.  Some are on sinfo.  This would be yet another place for them.  You should follow one of the other ways, or move the others to conf.